docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Live Context
+description: Powered by our Workstream Pattern Engine, Live Context enables the world’s first Temporally Grounded Copilot that understands your unique workflow
+---
+
+import {MiniSpacer} from "/src/components/Spacers";
+import Video from "/src/components/Video";
+
+# Applying Live Context to your Copilot
+
+:::info
+This feature is currently in Alpha and is now generally available to all users. It is only available to macOS & Windows users, with Linux support coming ASAP 🙏🏼.
+:::
+
+Since the original launch of Pieces for Developers, we’ve been laser-focused on developer productivity. We started by giving devs a place to store their most valuable snippets of code, then moved on to proactively saving and contextualizing them. Next, we built one of the first on-device LLM-powered AI copilots in the Pieces Copilot. Now, we’re taking our obsession with contextual developer productivity to the next level with the launch of Live Context within our copilot, making it the world’s first temporally grounded copilot.
+
+<Video type={'youtube'} src={'https://www.youtube.com/embed/aP8u95RTCGE?si=y4bhrmzG9KWsD1AS'}/>
+<MiniSpacer />
+
+Live Context elevates the Pieces Copilot and allows it to understand your workflow and discuss how, where, and when you work. As with everything we do here at Pieces, this all happens entirely on-device and is available on macOS, Windows, and Linux. We truly believe the Pieces Copilot+ will revolutionize how you cross off your to-dos.
```

Comment:
Also want to link to the blog Georgia just released

Reply:
Updated, let me know if you like where I linked the blog

Reply:
@tsavo-at-pieces